docs/ensembl-help/getting-started/species.md

```diff
@@ -9,10 +9,12 @@ tags:
 status: draft
 ---
 
-# Choose a species to work with
+# Choosing a species genome to work with
 
-Before using the other apps, you will need to use the Species selector:
+To start use the Species selector to select a species genome to work with in other apps:
 
 ![](../../img/id-species-selector.svg)
 
-Here you can choose the species you're working with, including specific genome assemblies and strains/breeds/varieties. You can have multiple species selected.
+Here you can choose the species genomes, assemblies, strains, breeds and varieties of interest to you. 
```

Comment:
There are no strains etc yet. Should we be setting expectations here ? How about:
"Here you can choose the species genomes and assemblies of interest to you. In the future this will be extended to strains, breeds and varieties."